apps/server/src/services/TimerService.ts

```diff
@@ -96,8 +96,8 @@ export class TimerService {
    * Loads roll information into timer service
    * @param {OntimeEvent[]} rundown -- list of events to run
    */
-  roll(rundown: OntimeEvent[]) {
-    runtimeState.roll(rundown);
+  roll(playableEvents: OntimeEvent[], rundown: OntimeRundown) {
```

Comment:
Question: I am not convinced this is something we should keep.

Historically we had problems with having the wrong dataset.

The playable events were supposed to be a cached, filtered version of the rundown.
At this point, I wonder if it is causing more issues than the convenience it brings.
This is especially true if we start using both the filtered (playableEvents) and unfiltered (rundown) datasets

I think the best is to extract utilities to filter the rundown and always pass the entire rundown around 

What do you think?

Reply:
yeah that is a bit harry.
Maybe we could use the Normalised Rundown 
and maybe extend the it with a list of timedEntrys 
like:
```ts
type RundownCache = {
  rundown: NormalisedRundown;
  order: string[];
  timedEvents: string[];
  revision: number;
  totalDelay: number;
  totalDuration: number;
};

``` 

or something like it?

Reply:
I am not sure. Let's chat